infertrade/utilities/operations.py

```diff
@@ -36,17 +37,19 @@
 
 def pct_chg(x: Union[np.ndarray, pd.Series]) -> np.ndarray:
     """Percentage change between the current and a prior element."""
-    x = x.astype("float64")
-
-    if isinstance(x, pd.DataFrame):
-        pc = x.pct_change().values.reshape(-1, 1)
+    if not isinstance(x, (pd.DataFrame, pd.Series, np.ndarray)):
+        raise TypeError("must be Pandas Series, DataFrame or numpy ndarray, not {}")
```

Comment:
Is there something missing in this line?

Reply:
@fernando-az-infertrade yes I've changed the code and committed.